Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Chip.h

```diff
@@ -146,6 +146,7 @@ class Chip : public TObject
     /// @param p Point to be added
     void InsertPoint(Point *p);
 
+    void SetGeometry(GeometryTGeo *g) { fGeometry=g; }
```

Comment:
Hello Matthias,

All your comments above are certainly valid.
But please, take into account that this is just a port from AliRoot, almost one-to-one.
It will take some extra time to bring it "up to the O2 standards".  And, I would prefer this to be done by the authors, since myself I am only helping with porting.
Thank you very much for your comments !  I will propagate them to the people in charge of these classes, and we will gradually improve.

Best regards,
Iouri.